src/main/registry.ts

```diff
@@ -0,0 +1,21 @@
+const regedit = require('regedit').promisified;
+
+const isPeaceInstalled = async () => {
+  const registryKey =
+    'HKLM\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall';
```

Comment:
Maybe we can define a file of useful constants later. I can see us putting this and the Peace hidden window in such a file. No rush to do that right away though.